pkgs/top-level/all-packages.nix

```diff
@@ -38711,4 +38711,7 @@ with pkgs;
   resgate = callPackage ../servers/resgate { };
 
   oversteer = callPackage ../applications/misc/oversteer { };
+
+  findex = callPackage ../applications/misc/findex { };
```

Comment:
This must be sorted.

Reply:
may I ask how?

Reply:
Just put this line in alphabetical order instead of at the end.